### PR TITLE
Set cookie on return-to-desktop

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @umbrae

### CODE REVIEW

File: assets/js/client.es6.js [hunk 1]
```diff
@@ -248,6 +248,11 @@ function initialize(bindLinks) {
   app.render(app.fullPathName(), true, modifyContext);
   app.config.renderTracking = true;
 
+  app.on('route:desktop', function(route) {
+    document.cookie = '__cf_mob_redir=0; expires=Fri, 31 Dec 2020 23:59:59 GMT';
```
Comment (umbrae) [inline assets/js/client.es6.js]: Someone's gonna be mad at you in 5 years. ;)

Comment (ajacksified) [inline assets/js/client.es6.js]: 98b39b21ff008afaa5cba1e79fc611b2ddb64b3f44f57e792397c517768f4433

**Comment by umbrae** [comment]
:fish: this codebase is cool

Only thought is you might want to make that cookie year dynamic. `(new Date()).getFullYear() + 1` or whatever the new hotness would be in ES6 isn't too hard.

**Comment by ajacksified** [comment]
Yeah, that's probably a good idea.

**Pull request merged**